psydac/ddm/cart.py

```diff
@@ -182,7 +182,11 @@ class DomainDecomposition:
         Number of threads used by one MPI rank.
 
     size: int
-        the size the communicator comm (=1 if comm is None).
+        The number of process assigned to the domain.
```

Comment:
You mean "processes" (plural).

Reply:
yes

Reply:
fixed in [6af4cad](https://github.com/pyccel/psydac/pull/232/commits/6af4cad9020d63885c2a35b7cf45b78414d2bf24)